tool/tsh/config.go

```diff
@@ -41,7 +42,7 @@ Host *.{{ .ClusterName }} {{ .ProxyHost }}
 # Flags for all {{ .ClusterName }} hosts except the proxy
 Host *.{{ .ClusterName }} !{{ .ProxyHost }}
     Port 3022
-    ProxyCommand "{{ .TSHPath }}" config-proxy --login="%r" --proxy={{ .ProxyHost }}:{{ .ProxyPort }} %h:%p "{{ .ClusterName }}"
+    ProxyCommand "{{ .TSHPath }}" proxy ssh --cluster={{ .ClusterName }} --proxy={{ .ProxyHost }} %r@%h:%p
```

Comment:
Will this work if `proxy_listener_mode` is not `multiplex`?

Also, why is `ProxyPort` no longer used?

Reply:
> Will this work if proxy_listener_mode is not multiplex?

yes, `tsh proxy ssh` logic  distinguish the `proxy_listener_mode` https://github.com/gravitational/teleport/pull/8920/files#diff-2ce402ac3a1c6b5da58e90affa61ed55008e021f83e2ba5f438fc7ec6af95e9aR57

> Also, why is ProxyPort no longer used?

`tsh proxy ssh`  obtains proxy configuration from the `~/.tsh/` proxy profile  so the port address is not longer needed.

Reply:
Really glad to see the port requirement go away!